app/views/my/page.html.erb

```diff
@@ -28,7 +28,7 @@ See doc/COPYRIGHT.rdoc for more details.
 ++#%>
 <% breadcrumb_paths(l(:label_my_page)) %>
 <% content_for :action_menu_specific do %>
-  <%= link_to l(:label_personalize_page), action: 'page_layout', accesskey: accesskey(:edit) %>
+  <%= link_to l(:label_personalize_page), { :action => 'page_layout', :accesskey => accesskey(:edit) }, "accesskey" => accesskey(:edit) %>
```

Comment:
specifying access key twice, once in the hash and ocne outside (what does that mean anyway?) looks suspicious...

Speculation: Either this change is not needed or the source of the problem lies elsewhere...